doc/classes/String.xml

```diff
@@ -704,6 +824,22 @@
 				For example, [code]"1,2.5,3"[/code] will return [code][1,2.5,3][/code] if split by [code]","[/code].
 			</description>
 		</method>
+		<method name="split_ints" qualifiers="const">
+			<return type="PackedInt32Array" />
+			<argument index="0" name="delimiter" type="String" />
+			<argument index="1" name="allow_empty" type="bool" default="true" />
+			<description>
+				Splits the string in integers by using a delimiter string and returns an array of the substrings.
+				For example, [code]"1,2.5,3"[/code] will return [code][1,0,3][/code] if split by [code]","[/code].
```

Comment:
That's a bit of a weird example to show (without explaining) what happens when one value is a float. I think it would be better to show what happens in a valid case, and then explain how non-integer substrings are treated.

Reply:
`rreplace` makes replacement starting from the right of the string. In a simple example:
```gdscript
"a a".replace("a", "B", 1) # prints "B a"
"a a".rreplace("a", "B", 1) # prints "a B"
```
In the above example, these calls are equivalents of `String.replace_first` and `String.replace_end` functionality-wise.

Reply:
Will fix in a future PR.